src/components/SelectionListWithModal/index.tsx

```diff
@@ -42,7 +42,8 @@ function SelectionListWithModal<TItem extends ListItem>(
     }, [sections, selectionMode, isSmallScreenWidth]);
 
     const handleLongPressRow = (item: TItem) => {
-        if (!turnOnSelectionModeOnLongPress || !isSmallScreenWidth) {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Why are we suppressing this here, and can we write the code in a way we don't need to suppress it?

Reply:
@neonbhai was the warning present before changing that line?